src/main/scala/ai/eto/rikai/RikaiOptions.scala

```diff
@@ -33,6 +33,19 @@ private[rikai] class RikaiOptions(parameters: Map[String, String]) {
     parameters
       .filterKeys(k => !RikaiOptions.excludedKeys(k))
       .toMap
+
+  /** Columns specified via df.partitionBy() */
+  val partitionColumns: Option[Seq[String]] =
+    parameters.get("__partition_columns") match {
+      case Some(cols) =>
+        Some(
+          cols
+            .substring(1, cols.length - 1)
```

Comment:
why do we need this `subtring` here?

Reply:
The value in `parameters: Map[string, string]` is something like

```
Map(__partition_columns -> ["label"], path -> /var/folders/m1/vyb5yj4n5cb17gv371g09k280000gn/T/rikai6565224676268032985/dataset)
```

the string value includes double quote